src/modules/tasksCompleted.js

```diff
@@ -0,0 +1,28 @@
+export default class CompletedTasksStatus {
+  completeTask = (array) => {
+    const todoTask = document.querySelectorAll('.todo-task');
+    const checkboxes = document.querySelectorAll('.check-task');
+    checkboxes.forEach((checkbox, index) => {
+      checkbox.addEventListener('click', () => {
+        todoTask[index].classList.toggle('active');
+        array[index].isCompleted = !array[index].isCompleted;
+        localStorage.setItem('tasks', JSON.stringify(array));
+        const todoInput = document.querySelectorAll('.todo-input');
+        if (todoTask[index].classList.contains('active')) {
+          todoInput[index].setAttribute('readOnly', true);
+        } else {
+          todoInput[index].removeAttribute('readOnly');
+        }
+      });
+    });
+  }
+
+  // delete all  completed tasks
+  clearCompleted = (tasks) => {
+    const clearBtn = document.querySelector('#completed-tasks');
+    clearBtn.addEventListener('click', () => {
+      tasks.tasksArray = tasks.tasksArray.filter((item) => item.isCompleted === false);
+      tasks.saveAndRender();
+    });
+  }
+}
```

Comment:
`README.md` file:
- [ ] I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**